rocks-db/src/columns/editions.rs

```diff
@@ -78,3 +109,39 @@ pub fn merge_token_metadata_edition(
 
     Some(result)
 }
+
+pub fn merge_token_metadata_parent_index_edition(
+    _new_key: &[u8],
+    existing_val: Option<&[u8]>,
+    operands: &MergeOperands,
+) -> Option<Vec<u8>> {
+    let mut result = vec![];
+    let mut write_version = 0;
+    if let Some(existing_val) = existing_val {
+        match serde_cbor::from_slice::<TokenMetadataEditionParentIndex>(existing_val) {
+            Ok(value) => {
+                write_version = value.write_version;
+                result = existing_val.to_vec();
+            },
+            Err(e) => {
+                error!("RocksDB: AssetCollection deserialize existing_val: {}", e)
+            },
+        }
+    }
+
+    for op in operands {
+        match serde_cbor::from_slice::<TokenMetadataEditionParentIndex>(op) {
+            Ok(new_val) => {
+                if new_val.write_version > write_version || result.is_empty() {
+                    write_version = new_val.write_version;
+                    result = op.to_vec();
+                }
```

Comment:
@StanChe as with MPL-core accounts, do we need to account for slot_updated here?

Reply:
Edition entity doesn't have slot information. I implemented the merge function in the same way as for Edition.
If we need check by slots, then this means that the current functionality for saving Edition works incorrectly and requires deeper refactoring.